sdks/go/pkg/beam/runners/dataflow/dataflowlib/job_test.go

```diff
@@ -157,3 +158,39 @@ func TestValidateWorkerSettings(t *testing.T) {
 		})
 	}
 }
+
+func TestCurrentStateMessage(t *testing.T) {
+	tests := []struct {
+		state   string
+		term    bool
+		want    string
+		wantErr error
+	}{
+		{state: "JOB_STATE_DONE", want: "Job JorbID-09876 succeeded", term: true},
```

Comment:
```suggestion
		{state: "JOB_STATE_DONE", want: "Job JorbID-09876 succeeded!", term: true},
```

Putting this here in case you want to quick commit both 😉